server/chat/backend/agent/utils/llm_usage_tracker.py

```
@@ -331,7 +331,19 @@ def store_usage(cls, usage: LLMUsage) -> bool:
+
+                # Report usage to marketplace metering (no-op unless hooks are configured).
+                # Implementations must be non-blocking (buffer + flush pattern).
+                try:
+                    from utils.hooks import get_hook
+                    get_hook("report_usage")(usage.org_id, usage.estimated_cost, {
+                        "model": usage.model_name,
+                        "input_tokens": usage.input_tokens,
+                        "output_tokens": usage.output_tokens,
+                    })
+                except Exception:
+                    pass
```

Ruff (0.15.17)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)